src/Altinn.App.Api/Controllers/LookupOrganisationController.cs

```diff
@@ -39,15 +39,15 @@ ILogger<OrganisationSearchController> logger
     /// Allows searching for an organisation in ER
     /// </summary>
     /// <param name="orgNr">Payload that contains params for executing a search for an organisation.</param>
```

Comment:
Nitpick: Description for orgNr is outdated. It references a payload with params inside, which we don't do here.